### ISSUE TITLE
Update Copyright

### ISSUE THREAD

**Issue by mattfarina** [kubernetes-helm/monocular#466]
Now that Monocular is under Helm in the CNCF they are advising we update the copyright to "Copyright The Helm Authors.". This is essentially replacing "Kubernetes" with "Helm" in the copyright statement for code files.

The CNCF is the parent organization for both Kubernetes and Helm. This is the direction coming from them.

**Comment by dankohn** [comment]
Specifically, the recommendation is to change from "Copyright 2016 The Kubernetes Authors All rights reserved." to "Copyright The Helm Authors." The regex to do that will need to take into account different years.

**Comment by dankohn** [comment]
By definition, all of the contributors to the Helm project to date constitute The Helm Authors. So, CNCF is advising that you can, in fact, change the copyright.

If you have concerns, please feel free to pull in Bitnami attorneys, who can discuss it with the CNCF attorneys.

**Comment by ridruejo** [comment]
Wow, that escalated quickly... I don't see an issue here, Bitnami is fine with the changes.

**Comment by sebgoa** [comment]
I am not speaking as Bitnami.

It was not clear to me from the "helm  proposal to CNCF" that all code under the `kubernetes-helm` organization where were part of the move.

We could use chart-museum as another example to avoid confusion.

But I will let this die

**Comment by dankohn** [comment]
@mattfarina For the record, could you please specify which repos will be moving to the new GitHub org? Any progress on that, BTW?

**Comment by sebgoa** [comment]
I went back to the proposal https://github.com/cncf/toc/blob/master/proposals/helm.adoc which clearly lists the projects part of the proposed moved.

It clearly states that monocular and chart-museum are part of the move, so I read the proposal too fast.

I do suggest that for a copyright change that you reference a voting email thread in this issue (or the PR that will actually make it happen).

**Comment by mattfarina** [comment]
For reference, the CNCF proposal had all things helm and charts related moving to the helm CNCF project.

@sebgoa If I understand it right, you're suggesting that in the PR for the copyright change we link to the CNCF proposal PR for full tractability? If so, I like that idea. If it's something else please correct me.

Note, Kubernetes and Helm are both under the CNCF. Changing the copyright doesn't change the parent organization.

**Comment by sebgoa** [comment]
@mattfarina yes, that's my suggestion. I think it is good practice to reference the discussion thread where this was voted/agreed, otherwise we loose track.
